core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -60,6 +60,14 @@ class TransactionStateManager(brokerId: Int,
 
   this.logIdent = "[Transaction Log Manager " + brokerId + "]: "
 
+  type WriteTxnMarkers = (String, // transactionalId
+    Long, // pid
+    Short, // producerEpoch
+    TransactionState, // currentState
+    TransactionMetadata, // current metadata
+    Int) // coordinatorEpoch
+    => Unit
```

Comment:
@ijuma How would a case class map these values to `Unit`? Or Do you mean a case class with an `apply` method defined in `TransactionCoordinator`?